src/config.rs

```diff
@@ -51,13 +51,25 @@ fn parse_cli_args() -> Result<CliArgs, getopts::Fail> {
     opts.optopt(
         "w",
         "width",
-        "Initial window's width in physical unit, the height command line arg much also be set",
+        "Initial window's width in physical unit, the height command line arg must also be set",
         "",
     );
     opts.optopt(
         "h",
         "height",
-        "Initial window's height in physical unit, the width command line arg much also be set",
+        "Initial window's height in physical unit, the width command line arg must also be set",
+        "",
+    );
+    opts.optopt(
+        "",
+        "x",
```

Comment:
Single character like `"x"` and `"y"` as long name seemed invalid.

```sh
$ cargo r
   Compiling versoview v0.0.1 (D:\Codes\verso)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 11.67s
     Running `target\debug\versoview.exe`
thread 'main' panicked at D:\packages\cargo\registry\src\index.crates.io-6f17d22bba15001f\getopts-0.2.21\src\lib.rs:604:5:
the long_name (second argument) should be longer than a single character, or an empty string for none
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: process didn't exit successfully: `target\debug\versoview.exe` (exit code: 101)
```

Reply:
Oops, tested the first width and height PR but rushed on this one, just opened a fix PR https://github.com/versotile-org/verso/pull/230